style.css

```diff
@@ -0,0 +1,372 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Mija, Helvetica, Arial, sans-serif;
+}
+
+nav {
+  display: flex;
+  background-color: #695e5c;
+  width: 100%;
+  height: 200px;
+  font-size: 500;
+  color: white;
+  padding-bottom: 50px;
+}
+
+.nav-images {
+  margin-right: 30px;
+}
+
+.icon-heading {
+  font-size: 25px;
+  font-family: Mija, Helvetica, Arial, sans-serif;
+}
+
+.icon-description {
+  font-size: 16.446px;
+  font-family: Elena, Helvetica, Arial, sans-serif;
+}
+
+.nav-main-images span {
+  float: right;
+  margin-top: 0;
+  position: relative;
+  top: 70px;
+  margin-right: 15px;
+}
+
+.nav-main-images img {
+  margin-top: 50px;
+  width: 100px;
+  height: 100px;
+}
+
+.nav-images img {
+  margin-top: 50px;
+  width: 60px;
+  height: 40px;
+}
+
+.nav-button {
+  margin-top: 100px;
+  margin-left: 90px;
+  width: 140px;
+}
+
+.nav-button button {
+  font-size: 30px;
+  font-family: Mija, Helvetica, Arial, sans-serif;
+  border: 4px solid white;
+  border-radius: 10px;
+  background-color: #695e5c;
+  color: white;
+}
+
+.fa-search {
+  color: white;
+  position: relative;
+  margin-right: 10px;
+  font-size: 20px;
+}
+
+.section1 {
+  display: grid;
+  grid-template-columns: auto auto;
+}
+
+.good-idea {
+  width: 100%;
+  height: 600px;
+  background-color: #9c9695;
+}
+
+.design {
+  width: 100%;
+  height: 600px;
+  background-color: #7d7c7c;
+}
+
+.embrace {
+  width: 100%;
+  height: 600px;
+  background-color: #52504f;
+}
+
+.seo {
+  width: 100%;
+  height: 600px;
+  background-color: #c4c4c4;
+}
```

Comment:
Similar styles in these classes should be combined under a single class and only the varying style should be specified separately for each of these classes.

Reply:
Sure.